core_lib/src/managers/playbackmanager.cpp

```diff
@@ -111,20 +111,6 @@ void PlaybackManager::play()
         frame = editor()->currentFrame();
     }
 
-    // get keyframe from layer
-    KeyFrame* key = nullptr;
-    if (!mListOfActiveSoundFrames.isEmpty())
-    {
-        for (int i = 0; i < object()->getLayerCount(); ++i)
-        {
-            Layer* layer = object()->getLayer(i);
-            if (layer->type() == Layer::SOUND)
-            {
-                key = layer->getKeyFrameWhichCovers(frame);
-            }
-        }
-    }
-
```

Comment:
Haven't checkout the branch with an IDE just quickly glanced through the code. Is there any reason to remove this section?

Reply:
No, but there isn’t any reason to keep it either as far as I can see. The value that’s computed here is never used afterwards.

Reply:
fair enough